src/Common/test-runtime/project.json

```diff
@@ -3,41 +3,21 @@
     "Microsoft.NETCore.Platforms": "1.0.1-rc4-24213-04",
     "Microsoft.NETCore.TestHost": "1.0.0-rc4-24131-00",
     "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc4-24213-03",
-    "NETStandard.Library": "1.6.0-rc4-24213-04",
```

Comment:
Which one of the dependencies is pulling in NETStandard.Library now? I kind of feel we should continue to explicitly reference it here so we can update and test the later versions.